server/src/main/java/com/tencent/rss/server/ShuffleServerFactory.java

```diff
@@ -23,8 +23,8 @@
 
 public class ShuffleServerFactory {
 
-  private final ShuffleServer shuffleServer;
-  private final ShuffleServerConf conf;
+  protected final ShuffleServer shuffleServer;
```

Comment:
`protected final` is not reasonable. `protected` means that you can override it. `final` means that you can't override it.